Core/src/main/java/com/plotsquared/core/plot/flag/GlobalFlagContainer.java

```diff
@@ -200,6 +201,7 @@ private GlobalFlagContainer() {
         this.addFlag(VillagerInteractFlag.VILLAGER_INTERACT_FALSE);
         this.addFlag(VineGrowFlag.VINE_GROW_TRUE);
         this.addFlag(ProjectilesFlag.PROJECTILES_FALSE);
+        this.addFlag(BeaconEffectFlag.BEACON_EFFECT_TRUE);
```

Comment:
If the necessary API doesn't exist on Spigot, the flag shouldn't be added to the global flag container, if the server runs Spigot.

Reply:
Unfortunately, that is the problem. Even if I like this flag idea.

Reply:
Oh, sry.

Reply:
> If the necessary API doesn't exist on Spigot, the flag shouldn't be added to the global flag container, if the server runs Spigot.

Is there a way to use paper specific flags? Because in the Core module you can't access the PaperLib library